__tests__/utils.test.js

```diff
@@ -28,21 +27,29 @@ describe("Northcoders News API", () => {
     });
 
     describe("/api/topics/", () => {
-      test("responds with an array of topic objects", () => {
+      test("200 - responds with an array of topic objects", () => {
         return request(app)
           .get("/api/topics")
           .expect(200)
           .then((response) => {
-            return response.body;
-          })
-          .then((data) => {
-            const expectedOutput = data.map((elem) => {
+            const expectedOutput = response.body.map((elem) => {
               expect(elem.hasOwnProperty("slug")).toBe(true);
               expect(elem.hasOwnProperty("description")).toBe(true);
             });
           });
       });
     });
+
+    describe("/api", () => {
+      test("200- respond with an object describing the available endpoints", () => {
+        return request(app)
+          .get("/api")
+          .expect(200)
+          .then((response) => {
+            expect(typeof response.body).toBe("object");
+          });
+      });
+    });
```

Comment:
So i would highly recommend reading through my hints here again: https://northcoders.slack.com/archives/C067A5EDC8L/p1705325400153309

This test is asserting almost nothing. We want to make sure the response matches the structure of the endpoints.json file that it's giving back. We can use JSON objects in our assertions. So what's stopping you from making a more specific assertion about the object this response gives you

Also please make sure you're sending data back on a key that describes it e.g. endpoints. Not directly on response.body